common/cpu/amd/pstate.nix

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I thinks this module should explain what it does.
Not everyone is a x86 expert. It would be also interesting to know, how this module differs from what Linux does on AMD by default.

Reply:
Upstream docs: https://www.kernel.org/doc/html/latest/admin-guide/pm/amd-pstate.html